perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/tabs/userstabs/SelfApplicationDetailTabItem.java

```diff
@@ -115,10 +116,26 @@ public Widget draw() {
 		vp.add(sp);
 		vp.setCellHeight(sp, "100%");
 
-		GetApplicationDataById data = new GetApplicationDataById(applicationId);
-		data.setShowAdminItems(false);
-		data.retrieveData();
-		sp.setWidget(data.getContents());
+		if (application.getType().equalsIgnoreCase("EMBEDDED")) {
+			final FlexTable ft = new FlexTable();
+			ft.setCellSpacing(5);
+			ft.setHTML(0, 0, "<strong>Name:</strong> " + SafeHtmlUtils.fromString(application.getUser().getFullNameWithTitles()).asString());
```

Comment:
Couldn't there be a problem when there is no name and titles?

Reply:
I haven't thought of that, I have added a check.